datumaro/plugins/transforms.py

```diff
@@ -948,12 +987,18 @@ class RemoveAttributes(ItemTransform):
     Can be useful to clean the dataset from broken or unnecessary attributes.|n
     |n
     Examples:|n
-    - Remove the `is_crowd` attribute from dataset|n
-    |s|s%(prog)s --attr 'is_crowd'|n
-    |n
-    - Remove the `occluded` attribute from annotations of|n
-    |s|sthe `2010_001705` item in the `train` subset|n
-    |s|s%(prog)s --id '2010_001705:train' --attr 'occluded'
+        - Remove the `is_crowd` attribute from dataset|n
+
+        .. code-block::
+
+        |s|s%(prog)s --attr 'is_crowd'|n
+        |n
+        - Remove the `occluded` attribute from annotations of|n
+        |s|sthe `2010_001705` item in the `train` subset|n
+
+        .. code-block::
+
+        |s|s%(prog)s --id '2010_001705:train' --attr 'occluded'
```

Comment:
```suggestion
        - Remove the `is_crowd` attribute from dataset|n

        .. code-block::

        |s|s%(prog)s --attr 'is_crowd'|n
        |n
        - Remove the `occluded` attribute from annotations of|n
        |s|sthe `2010_001705` item in the `train` subset|n

        .. code-block::

        |s|s%(prog)s --id '2010_001705:train' --attr 'occluded'
```
This formatting will be lost in CLI.

Reply:
directives `.. code-block::` and empty lines will be removed in the CLI
![image](https://user-images.githubusercontent.com/54434686/155140002-f788674d-7515-461d-8d3f-a094e15273a2.png)

Reply:
Yes, but I'm saying about the added indentation level in the examples:
```
    ...
    |n
    Examples:|n
        - Remove the `is_crowd` attribute from dataset|n
    ...
```
Can we remove extra indents?

Reply:
In this case, the indentation is made to correctly display  in the API documentation, so that this indentation is not expected in the CLI.  Maybe replace the indentation with `|s|s|s|s` in the docstrings which are displayed in CLI?

Reply:
No, I don't see any point in extra indentation for examples.